docs/medical-api/api-reference/patient/search-patient.mdx

```diff
@@ -0,0 +1,56 @@
+---
+title: "Search Patient"
+description: "Searches for a Patient in Metriport based on demographic data. Returns a 404 if the patient is not found."
```

Comment:
Note that devs may get confused if they assume this will search the HIEs for the patient - perhaps we can clarify this by stating:

`Searches for a Patient previously created in Metriport, based on demographic data. Returns a 404 if the patient is not found.`

Reply:
Done.